app.py

```diff
@@ -107,12 +107,12 @@ def before_request_modifier():
     def inject_global_constants():
         return dict(
             stage="beta",
-            service_title="Funding Service Design - Authenticator",
+            service_title="Apply for funding to save a building in your community",
```

Comment:
you can add global config here instead of injecting it into every view. These should be accessible everywhere.